### ISSUE TITLE
wlr-gamma-control changes deferred for fullscreen application

### ISSUE THREAD

**Issue by Xyene** [swaywm/wlroots#2280]
If an application is fullscreen on one output, gamma changes on that output don't seem to be applied until the application leaves fullscreen mode. A simple reproduction seems to be fullscreening an application, then starting up the Wayland Redshift fork, and noticing the gamma change is not applied for the output with the fullscreen application *until* fullscreen is exited.

(I suppose this could be an issue in Redshift itself, but a quick read through the code didn't show anything related.)

I'm using amdgpu drivers (5.4.0 kernel) with an RX 480. Curiously, this only seems to be happening on one of my outputs (1440p@144Hz, `adaptive_sync` off, works fine on my other 1440p@60Hz), and there is nothing printed to the log during the gamma switch.

<details><summary>Startup logs</summary>

```
00:00:00.000 [sway/main.c:153] Linux lynwood 5.4.0-0.bpo.3-amd64 #1 SMP Debian 5.4.13-1~bpo10+1 (2020-02-07) x86_64 GNU/Linux
00:00:00.000 [sway/main.c:169] Contents of /etc/os-release:                                                             
00:00:00.000 [sway/main.c:153] PRETTY_NAME="Debian GNU/Linux 10 (buster)"                                               
00:00:00.000 [sway/main.c:153] NAME="Debian GNU/Linux"                                                                  
00:00:00.000 [sway/main.c:153] VERSION_ID="10"                                                                          
00:00:00.000 [sway/main.c:153] VERSION="10 (buster)"                                                                    
00:00:00.000 [sway/main.c:153] VERSION_CODENAME=buster                                                                  
00:00:00.000 [sway/main.c:153] ID=debian                                                                                
00:00:00.000 [sway/main.c:153] HOME_URL="https://www.debian.org/"                                                       
00:00:00.000 [sway/main.c:153] SUPPORT_URL="https://www.debian.org/support"                                             
00:00:00.000 [sway/main.c:153] BUG_REPORT_URL="https://bugs.debian.org/"                                                
00:00:00.000 [sway/main.c:169] Contents of /etc/debian_version:                                                         
00:00:00.000 [sway/main.c:153] 10.3                                                                                     
00:00:00.000 [sway/main.c:141] LD_LIBRARY_PATH=(null)                                                                   
00:00:00.000 [sway/main.c:141] LD_PRELOAD=(null)                                                                        
00:00:00.000 [sway/main.c:141] PATH=/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games:/home/xyene/.dotnet/tools  
00:00:00.000 [sway/main.c:141] SWAYSOCK=(null)                                                                          
00:00:00.000 [sway/server.c:42] Preparing Wayland server initialization                                                 
00:00:00.002 [INFO] [backend/session/logind.c:822] Successfully loaded logind session                                   
00:00:00.003 [INFO] [backend/backend.c:157] Found 1 GPUs                                                                
00:00:00.003 [INFO] [backend/drm/backend.c:138] Initializing DRM backend for /dev/dri/card0 (amdgpu)                    
00:00:00.003 [DEBUG] [backend/drm/drm.c:70] Using atomic DRM interface                                                                                                                                                                                                                                                      
00:00:00.003 [INFO] [backend/drm/drm.c:255] Found 6 DRM CRTCs                                                           
00:00:00.003 [INFO] [backend/drm/drm.c:176] Found 12 DRM planes                                                         
00:00:00.066 [INFO] [render/egl.c:246] Using EGL 1.5                                                                    
00:00:00.066 [INFO] [render/egl.c:247] Supported EGL extensions: EGL_ANDROID_native_fence_sync EGL_EXT_buffer_age EGL_EXT_create_context_robustness EGL_EXT_image_dma_buf_import EGL_KHR_cl_event2 EGL_KHR_config_attribs EGL_KHR_create_context EGL_KHR_create_context_no_error EGL_KHR_fence_sync EGL_KHR_get_all_proc_add
00:00:00.066 [INFO] [render/egl.c:248] EGL vendor: Mesa Project                                                         
00:00:00.066 [DEBUG] [render/egl.c:127] Supported dmabuf buffer formats: AR24 XR24                                      
00:00:00.068 [INFO] [render/gles2/renderer.c:581] Using OpenGL ES 3.2 Mesa 18.3.6                                       
00:00:00.068 [INFO] [render/gles2/renderer.c:582] GL vendor: X.Org                                                      
00:00:00.068 [INFO] [render/gles2/renderer.c:583] GL renderer: AMD Radeon (TM) RX 480 Graphics (POLARIS10, DRM 3.35.0, 5.4.0-0.bpo.3-amd64, LLVM 7.0.1)
00:00:00.068 [INFO] [render/gles2/renderer.c:584] Supported GLES2 extensions: GL_EXT_blend_minmax GL_EXT_multi_draw_arrays GL_EXT_texture_filter_anisotropic GL_EXT_texture_compression_s3tc GL_EXT_occlusion_query_boolean GL_EXT_texture_compression_dxt1 GL_EXT_texture_format_BGRA8888 GL_OES_compressed_ETC1_RGB8_textu
00:00:00.068 [DEBUG] [GLES2] Shader Stats: SGPRS: 16 VGPRS: 8 Code Size: 208 LDS: 0 Scratch: 0 Max Waves: 8 Spilled SGPRs: 0 Spilled VGPRs: 0 PrivMem VGPRs: 0
00:00:00.068 [DEBUG] [GLES2] Shader Stats: SGPRS: 8 VGPRS: 24 Code Size: 64 LDS: 0 Scratch: 0 Max Waves: 8 Spilled SGPRs: 0 Spilled VGPRs: 0 PrivMem VGPRs: 0
00:00:00.068 [DEBUG] [GLES2] Shader Stats: SGPRS: 16 VGPRS: 12 Code Size: 236 LDS: 0 Scratch: 0 Max Waves: 8 Spilled SGPRs: 0 Spilled VGPRs: 0 PrivMem VGPRs: 0
00:00:00.069 [DEBUG] [GLES2] Shader Stats: SGPRS: 8 VGPRS: 24 Code Size: 140 LDS: 0 Scratch: 0 Max Waves: 8 Spilled SGPRs: 0 Spilled VGPRs: 0 PrivMem VGPRs: 0
00:00:00.069 [DEBUG] [GLES2] Shader Stats: SGPRS: 24 VGPRS: 8 Code Size: 224 LDS: 0 Scratch: 0 Max Waves: 8 Spilled SGPRs: 0 Spilled VGPRs: 0 PrivMem VGPRs: 0
00:00:00.069 [DEBUG] [GLES2] Shader Stats: SGPRS: 24 VGPRS: 24 Code Size: 136 LDS: 0 Scratch: 0 Max Waves: 8 Spilled SGPRs: 0 Spilled VGPRs: 0 PrivMem VGPRs: 0
00:00:00.069 [DEBUG] [GLES2] Shader Stats: SGPRS: 24 VGPRS: 24 Code Size: 140 LDS: 0 Scratch: 0 Max Waves: 8 Spilled SGPRs: 0 Spilled VGPRs: 0 PrivMem VGPRs: 0
00:00:00.069 [DEBUG] [GLES2] Shader Stats: SGPRS: 24 VGPRS: 8 Code Size: 224 LDS: 0 Scratch: 0 Max Waves: 8 Spilled SGPRs: 0 Spilled VGPRs: 0 PrivMem VGPRs: 0
00:00:00.069 [DEBUG] [GLES2] Shader Stats: SGPRS: 24 VGPRS: 24 Code Size: 136 LDS: 0 Scratch: 0 Max Waves: 8 Spilled SGPRs: 0 Spilled VGPRs: 0 PrivMem VGPRs: 0
00:00:00.069 [DEBUG] [GLES2] Shader Stats: SGPRS: 24 VGPRS: 8 Code Size: 224 LDS: 0 Scratch: 0 Max Waves: 8 Spilled SGPRs: 0 Spilled VGPRs: 0 PrivMem VGPRs: 0
...
00:00:00.484 [DEBUG] [backend/libinput/backend.c:101] libinput successfully initialized
00:00:00.484 [INFO] [backend/drm/drm.c:1241] Scanning DRM connectors
00:00:00.484 [INFO] [backend/drm/drm.c:1300] Found connector 'DP-1'
00:00:00.484 [INFO] [backend/drm/drm.c:1336] 'DP-1' connected
00:00:00.484 [DEBUG] [backend/drm/drm.c:1338] Current CRTC: 47
00:00:00.484 [INFO] [backend/drm/drm.c:1343] Physical size: 600x340
00:00:00.484 [INFO] [backend/drm/drm.c:1360] Detected modes:
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   2560x1440@144000 (preferred)
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   2560x1440@120000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   2560x1440@99900 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   2560x1440@61029 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   2560x1440@59951 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1920x1200@144000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1920x1080@74977 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1920x1080@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1920x1080@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1920x1080@59940 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1920x1080@50000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1600x1200@144000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1680x1050@59954 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1600x900@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1280x1024@75025 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1280x1024@60020 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1440x900@144000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1280x800@59810 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1152x864@59972 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1280x720@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1280x720@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1280x720@59940 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1280x720@50000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1024x768@75029 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1024x768@60004 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   800x600@75000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   800x600@60317 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   720x480@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   720x480@59940 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   640x480@75000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   640x480@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   640x480@59940 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   640x480@59940 
00:00:00.484 [INFO] [backend/drm/drm.c:1300] Found connector 'DP-2'
00:00:00.484 [INFO] [backend/drm/drm.c:1336] 'DP-2' connected
00:00:00.484 [DEBUG] [backend/drm/drm.c:1338] Current CRTC: 49
00:00:00.484 [INFO] [backend/drm/drm.c:1343] Physical size: 600x340
00:00:00.484 [INFO] [backend/drm/drm.c:1360] Detected modes:
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   2560x1440@59951 (preferred)
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1920x1440@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1856x1392@59995 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1792x1344@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   2048x1152@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1920x1200@74930 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1920x1200@59885 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1920x1080@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1920x1080@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1920x1080@59940 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1920x1080@50000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1920x1080@50000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1920x1080@30000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1920x1080@29970 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1920x1080@25000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1920x1080@24000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1920x1080@23976 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1600x1200@75000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1600x1200@70000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1600x1200@65000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1600x1200@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1680x1050@74892 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1680x1050@59954 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1680x945@60017 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1400x1050@74867 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1400x1050@59978 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1600x900@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1280x1024@75025 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1280x1024@60020 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1440x900@74984 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1440x900@59887 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1280x960@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1366x768@59790 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1360x768@60015 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1280x800@74934 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1280x800@59810 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1152x864@75000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1280x768@74893 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1280x768@59870 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1280x720@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1280x720@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1280x720@59940 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1280x720@50000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1440x576@50000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1440x576@50000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1024x768@75029 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1024x768@70069 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1024x768@60004 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1440x480@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1440x480@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1440x480@59940 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1440x480@59940 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   1024x576@59967 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   832x624@74551 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   800x600@75000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   800x600@72188 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   800x600@60317 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   800x600@56250 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   720x576@50000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   720x576@50000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   848x480@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   720x480@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   720x480@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   720x480@59940 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   720x480@59940 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   720x480@59940 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   640x480@75000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   640x480@72809 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   640x480@66667 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   640x480@60000 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   640x480@59940 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   640x480@59940 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   720x400@70082 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   720x240@60117 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   720x240@60117 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   720x240@60054 
00:00:00.484 [INFO] [backend/drm/drm.c:1385]   720x240@60054 
00:00:00.484 [INFO] [backend/drm/drm.c:1300] Found connector 'HDMI-A-1'
00:00:00.484 [INFO] [backend/drm/drm.c:1300] Found connector 'HDMI-A-2'
00:00:00.484 [INFO] [backend/drm/drm.c:1300] Found connector 'DVI-D-1'
00:00:00.484 [DEBUG] [backend/drm/drm.c:1073] Reallocating CRTCs
00:00:00.484 [DEBUG] [backend/drm/drm.c:1084] State before reallocation:
00:00:00.484 [DEBUG] [backend/drm/drm.c:1093]   'DP-1' crtc=0 state=1 desired_enabled=1
00:00:00.484 [DEBUG] [backend/drm/drm.c:1093]   'DP-2' crtc=1 state=1 desired_enabled=1
00:00:00.484 [DEBUG] [backend/drm/drm.c:1093]   'HDMI-A-1' crtc=-1 state=0 desired_enabled=0
00:00:00.484 [DEBUG] [backend/drm/drm.c:1093]   'HDMI-A-2' crtc=-1 state=0 desired_enabled=0
00:00:00.485 [DEBUG] [backend/drm/drm.c:1093]   'DVI-D-1' crtc=-1 state=0 desired_enabled=0
00:00:00.485 [DEBUG] [backend/drm/drm.c:1142] State after reallocation:
00:00:00.485 [DEBUG] [backend/drm/drm.c:1152]   'DP-1' crtc=0 state=1 desired_enabled=1
00:00:00.485 [DEBUG] [backend/drm/drm.c:1152]   'DP-2' crtc=1 state=1 desired_enabled=1
00:00:00.485 [DEBUG] [backend/drm/drm.c:1152]   'HDMI-A-1' crtc=-1 state=0 desired_enabled=0
00:00:00.485 [DEBUG] [backend/drm/drm.c:1152]   'HDMI-A-2' crtc=-1 state=0 desired_enabled=0
00:00:00.485 [DEBUG] [backend/drm/drm.c:1152]   'DVI-D-1' crtc=-1 state=0 desired_enabled=0
00:00:00.485 [INFO] [backend/drm/drm.c:1449] Requesting modeset for 'DP-1'
00:00:00.485 [sway/desktop/output.c:923] New output 0x55e76a312b30: DP-1
00:00:00.485 [sway/config/output.c:348] Turning on output DP-1
00:00:00.485 [sway/config/output.c:353] Set DP-1 mode to 2560x1440 (-1.000000 Hz)
00:00:00.485 [sway/config/output.c:266] Assigning configured mode to DP-1
00:00:00.485 [sway/config/output.c:326] Output DPI: 108.373333x107.576471
00:00:00.485 [sway/config/output.c:379] Auto-detected output scale: 1.000000
00:00:00.485 [sway/config/output.c:418] Committing output DP-1
00:00:00.485 [INFO] [backend/drm/drm.c:805] Modesetting 'DP-1' with '2560x1440@144000 mHz'
00:00:00.485 [DEBUG] [backend/drm/drm.c:686] Initializing renderer on connector 'DP-1'
00:00:00.485 [DEBUG] [GLES2] Shader Stats: SGPRS: 8 VGPRS: 28 Code Size: 36 LDS: 0 Scratch: 0 Max Waves: 8 Spilled SGPRs: 0 Spilled VGPRs: 0 PrivMem VGPRs: 0
00:00:00.485 [DEBUG] [GLES2] Shader Stats: SGPRS: 24 VGPRS: 12 Code Size: 276 LDS: 0 Scratch: 0 Max Waves: 8 Spilled SGPRs: 0 Spilled VGPRs: 0 PrivMem VGPRs: 0
00:00:00.485 [DEBUG] [GLES2] Shader Stats: SGPRS: 8 VGPRS: 24 Code Size: 40 LDS: 0 Scratch: 0 Max Waves: 8 Spilled SGPRs: 0 Spilled VGPRs: 0 PrivMem VGPRs: 0
00:00:00.485 [DEBUG] [GLES2] Shader Stats: SGPRS: 16 VGPRS: 8 Code Size: 108 LDS: 0 Scratch: 0 Max Waves: 8 Spilled SGPRs: 0 Spilled VGPRs: 0 PrivMem VGPRs: 0
00:00:00.486 [DEBUG] [GLES2] LLVM diagnostic (remark): <unknown>:0:0: 4 instructions in function
```

**Comment by emersion** [comment]
Would be useful to add debug `wlr_log` calls to the gamma-control impl and to the DRM backend.

**Comment by project-repo** [comment]
Hi,
I came across this issue recently in trying to fix  gamma-control for `cage` and `cagebreak` and can in fact still reproduce this. The problem seems to be that the redshift is not applied when using `wlr_output_attach_buffer` to render a fullscreen view, as disabling the scanning out of full screen views results in redshift working even when the current view is fullscreen. If you need any additional information, don't hesitate to ask!
cheers,
project-repo

**Comment by Xyene** [comment]
I haven't gotten around to adding the requested logs, but did want to update that the behavior of this bug has changed in recent wlroots to be much less annoying. Presently, starting e.g. `gammastep -O 1000` immediately applies the gamma to all displays, regardless of if they have fullscreen applications running.

Quitting gammastep still defers the resetting of the gamma until at least one of:

* the cursor image changes
* (in the case of multiple outputs) an inactive output is focused

This is again better than before, where one would have to leave fullscreen in order for the gamma to be applied.

**Comment by project-repo** [comment]
I can actually still reproduce the original manifestation of the bug in that the gamma is not applied when an application is fullscreen... My guess is that since the latest update the application you are using to test this is no longer scanned out.

**Comment by project-repo** [comment]
This issue seems to be fixed in wlroots 0.13.0, at least in cagebreak.
Use `(sleep 5&&gammastep -O 1000)&(sleep 10&&killall gammastep)` to reproduce.
I recommend to close this issue.
cheers
project-repo

**Comment by Xyene** [comment]
Yes, this seems fixed for me too.
